src/index.html

```diff
@@ -11,6 +11,38 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form class="searchBar" data-qa="big">
```

Comment:
Consider using more descriptive class names instead of `searchBar`. For example, `searchForm` or `searchContainer`. 🤔